default_warehouse_from_sale_team/i18n/es.po

```diff
@@ -219,23 +230,16 @@ msgid "Stock Move"
 msgstr "Movimiento de existencias"
 
 #. module: default_warehouse_from_sale_team
-#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_res_users__sale_team_ids
-msgid ""
-"This is only an informative field. Go to Sales > Sales > Sales Teams to edit"
-msgstr ""
-"Esto es sólo un campo informativo. Vaya a Ventas> Ventas> Equipos de ventas "
-"para editar"
+#: code:addons/default_warehouse_from_sale_team/models/res_users.py:0
+#, python-format
+msgid "The chosen team (%s) is not in the allowed sales teams for this user"
+msgstr "El equipo elegido (%s) no está entre los equipos de ventas permitidos para este usuario"
```

Comment:
```suggestion
msgstr "El equipo de venta seleccionado (%s) no está entre los equipos permitidos para este usuario"
```

Based on [this](https://github.com/odoo/odoo/blob/210913ad8475d543bf04a3fce9a9616ab879cce1/odoo/addons/base/i18n/es.po#L28899)

Reply:
Change done 😄